pkg/policy/validate.go

```diff
@@ -76,23 +77,23 @@ func validateJSONPatchPathForForwardSlash(patch string) error {
 }
 
 // Validate checks the policy and rules declarations for required configurations
-func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool, openAPIController *openapi.Controller) error {
+func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool, openAPIController *openapi.Controller) (*v1beta1.AdmissionResponse, error) {
```

Comment:
IMHO, we should also remove the first return value if it's not used anywhere. We can add it when needed.

Reply:
Removed for now.